custom_components/waste_collection_schedule/waste_collection_schedule/source/region_gotland_se.py

```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+from datetime import datetime
+import json
+from urllib.parse import urlencode
+
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Region Gotland"
+DESCRIPTION = "Source for Region Gotland waste collection."
+URL = "https://edpfuture.gotland.se"
+TEST_CASES = {
+    "Dummy": {"uprn": "0000000000"},
+}
+
+class Source:
+    def __init__(self, uprn):
+        self._uprn = uprn
+
+    def fetch(self):
+        query_params = urlencode({"address": '(' + self._uprn + ')'})
+        response = requests.get(
```

Comment:
Please pass arguments in URLs via the `params` argument. This will probably obsolete the `urlencode` above.